syntax="proto3";
package IM;
message ChatMsg
{
      int32 networkType=1;
      int32 msgID=2;        //每条消息的标识
      int32 recvID=3;       //消息接收者
      int32 sendID=4;
      bytes sendName=5;
      string sendtime=6;
      bytes content=7;
}